src/image.cpp

```diff
@@ -52,6 +52,7 @@ namespace librealsense
         case RS2_FORMAT_Z16H: return 16;
         case RS2_FORMAT_FG: return 16;
         case RS2_FORMAT_Y411: return 12;
+        case RS2_FORMAT_Y16I: return 32;
```

Comment:
Can you please replace the Y12i comment?
We need to drop D457 dev comments with time.
` was 24 for UVC, D457 GMSL SerDes require 32 so we increased it for both` 
What do you think?

Reply:
sounds ok